semeio/jobs/rft/zonemap.py

```diff
@@ -88,14 +88,14 @@ def load_and_parse_zonemap_file(cls, filename):
     def __contains__(self, item):
         if isinstance(item, int):
             return item in self._zones_at_k_value
-        elif isinstance(item, str):
+        if isinstance(item, str):
             return item in self._k_values_at_zone
         return False
 
     def __getitem__(self, item):
         if isinstance(item, int):
             return self._zones_at_k_value[item]
-        elif isinstance(item, str):
+        if isinstance(item, str):
```

Comment:
Why is `elif` wrong here?

Reply:
It is unnecessary since you are returning in the if statement above. Pylint prefers that you do not continue with elif/else if you return in the first if statement.